File: ads3/poo/Semana2/aula02/c++/Main.cpp
#include <iostream>
using namespace std;

bool primo(int num){
	for(int i = 1,cont=0; i<=num; i++){
		if(num%i == 0)
			cont++;
				
		if(cont>2 || num == 1)
			return false;
	}
	return true;
}
	
int main(int argc, char *argv[]){
	int valor;
	cin >> valor;
	
	if(primo(valor))
		cout << "eh um numero primo" << endl;
	else
		cout << "nao eh um numero primo" << endl;
			
		
	return 0;
}

File: ads3/poo/Semana2/aula02/java/Main.java
import java.util.Scanner;
public class Main{
	public static boolean primo(int num){
		for(int i = 1,cont=0; i<=num; i++){
			if(num%i == 0)
				cont++;
				
			if(cont>2 || num == 1)
				return false;
		}
		return true;
	}
	
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		int valor = sc.nextInt();
		sc.close();
		
		if(primo(valor))
			System.out.println("eh um numero primo");
		else
			System.out.println("nao eh um numero primo");
		
	}
}
